src/Spd.Manager.Shared/SharedManagerFuncs.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Spd.Resource.Repository;
+using Spd.Resource.Repository.Org;
+using Spd.Resource.Repository.User;
+using Spd.Utilities.Shared.Exceptions;
+using System.Net;
+
+namespace Spd.Manager.Shared;
+public static class SharedManagerFuncs
+{
+    public static void CheckMaxRoleNumberRuleAsync(OrgQryResult org, List<UserResult> userList)
```

Comment:
we cannot use this as for biz, the role is PrimaryManager not Primary.

Reply:
Fixed.